src/IdentityManager/Repository.EntityFramework/Migrations/20210308183812_init.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using System;
 using Microsoft.EntityFrameworkCore.Migrations;
 
-namespace Travely.IdentityManager.Repository.Migrations
+namespace Travely.IdentityManager.Repository.EntityFramework.Migrations
 {
-    public partial class CreateIdentityServerDB : Migration
+    public partial class init : Migration
```

Comment:
Recreate migration and made it capital `init` -> `Init`, so we match C# code style in generated classes.